src/styles.scss

```diff
@@ -1 +1,36 @@
 /* You can add global styles to this file, and also import other style files */
+.flex {
+  display: flex;
+}
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+table { width: 100%;}
+tr.data-row:hover { 
+  background-color: grey;
+}
+
+.product-list-component {
```

Comment:
these classes where they are specific to a certain place can be moved to where they are used (so like product-list-component, product-detail-component, detail-radio-group, detail-radio-button) or some of them could have more generic names if you're going to be re-using them